### ISSUE TITLE
Upload Method

### ISSUE THREAD

**Issue by sstaub** [platformio/platform-wiznet7500#1]
It seems that the upload works with copy the firmware file to the mbed disc. On a Mac you
get some message because of unregulary unmounting the board.
It would be better to upload directly via cmsis-dap? like on the mbed lpc1768 board,
this would avoid the unmounting messages of the mbed disc.

**Comment by ivankravets** [comment]
Do you experience a similar issue with other mbed-enabled boards?

**Comment by sstaub** [comment]
Yes, longer times ago (more than a year) I have the same issue with mbed lpc1768 board,
but you found a solution with the upload method, not using the mbed disc.

stefan

**Comment by ivankravets** [comment]
We use the same method (copying to media disk) for NXP LPC, see https://github.com/platformio/platform-nxplpc/blob/develop/builder/main.py#L138

It seems that disk disappears when new firmware is copied. Honestly, that is correct behavior. Board goes to reboot...

**Comment by sstaub** [comment]
I tested the same code with my mbed lpc 1768 board, and there is no warning about incorrect unmounting, curious.
I see that the upload code is the same. Another difference is that the wiznet board resets automatic, the mbed lpc1768
needs a manual reset, really curious. I can live with that issue.

stefan
